meson.build

```diff
@@ -437,35 +437,39 @@ configure_file(input : 'dbus/org.freedesktop.ratbag_devel1.conf.in',
 	       configuration : config_ratbagd_devel)
 
 #### unit file ####
-unitdir = get_option('systemd-unit-dir')
-if unitdir == ''
-	libdir = get_option('libdir')
-	default_unitdir = dep_systemd.get_pkgconfig_variable('systemdsystemunitdir')
-	# Fedora uses lib64 but systemd is in lib. Hack around this so it
-	# works out of the box.
-	intended_unitdir = join_paths(get_option('prefix'), get_option('libdir'), 'systemd')
-	if get_option('prefix') == '/usr' and intended_unitdir != default_unitdir
-		message('''
-		systemd unitdir libdir mismatch detected, changing unitdir to
-			@0@
-		or specify with
-			mesonconf -Dsystemd-unit-dir=<path>
-
-		See https://github.com/libratbag/libratbag/issues/188
-		'''.format(default_unitdir))
-		unitdir = default_unitdir
-	else
-		unitdir = intended_unitdir
+if dep_systemd.found()
+	unitdir = get_option('systemd-unit-dir')
+	if unitdir == ''
+		libdir = get_option('libdir')
+		default_unitdir = dep_systemd.get_pkgconfig_variable('systemdsystemunitdir')
+		# Fedora uses lib64 but systemd is in lib. Hack around this so it
+		# works out of the box.
+		intended_unitdir = join_paths(get_option('prefix'), get_option('libdir'), 'systemd')
+		if get_option('prefix') == '/usr' and intended_unitdir != default_unitdir
+			message('''
+			systemd unitdir libdir mismatch detected, changing unitdir to
+				@0@
+			or specify with
+				mesonconf -Dsystemd-unit-dir=<path>
+
+			See https://github.com/libratbag/libratbag/issues/188
+			'''.format(default_unitdir))
+			unitdir = default_unitdir
+		else
+			unitdir = intended_unitdir
+		endif
 	endif
 endif
 
 config_bindir = configuration_data()
 config_bindir.set('bindir', join_paths(get_option('prefix'), get_option('bindir')))
 
```

Comment:
are we using config_bindir anywhere else? If not, we can move it into the if condition, then you don't need to check for `dep_systemd.found()` twice.

Reply:
True. Has to be fixed in master then.

Reply:
nah, doesn't need to be fixed, just checked and it's used for the dbus service file too